velox/dwio/parquet/tests/reader/ParquetReaderTest.cpp

```diff
@@ -632,3 +632,60 @@ TEST_F(ParquetReaderTest, preloadSmallFile) {
     ASSERT_EQ(file->bytesRead(), 0);
   }
 }
+
+TEST_F(ParquetReaderTest, prefetchRowGroups) {
+  auto rowType = ROW({"id"}, {BIGINT()});
+  const std::string sample(getExampleFilePath("multiple_row_groups.parquet"));
+  const int numRowGroups = 4;
+
+  facebook::velox::dwio::common::ReaderOptions readerOptions{defaultPool.get()};
+  // Disable preload of file.
+  readerOptions.setFilePreloadThreshold(0);
```

Comment:
should we also set directorySizeGuess with a small size? such as 1024, 
 kDefaultDirectorySizeGuess is 1MB.

```c++
  bool preloadFile =
      fileLength_ <= std::max(filePreloadThreshold_, directorySizeGuess_);
```